usage/jsgui/src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -33,6 +33,11 @@ define([
         return _.escape(s);
     };
 
+    Util.unescape = function (s) {
+        if (s == undefined || s == null) return "";
```

Comment:
The preferred way is to use `typeof` and compare against the string `'undefined'`:

http://stackoverflow.com/questions/3390396/how-to-check-for-undefined-in-javascript